ydb/core/kqp/ut/join/data/queries/tpcds34.sql

```diff
@@ -0,0 +1,38 @@
+pragma TablePathPrefix = "/Root/test/ds/";
```

Comment:
I'd suggest to add a minimal query that reproduces the bug being fixed and give a reference to this tpc-h query

Reply:
Added a query to KqpOlap unit test.
Keeping this query though - let's make sure we check all benchmark queries that have failed so far